package.json

```diff
@@ -49,6 +53,7 @@
     "*.vue": [
       "vue-cli-service lint",
       "git add"
-    ]
+    ],
+    "**/*": "prettier --write --ignore-unknown"
```

Comment:
Installation guide suggests this. Its linked to git-hooks, which I'm assuming vuCLI setup on our behalf, as this section was already there.